sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasQueryParameters.cs

```diff
@@ -0,0 +1,129 @@
+﻿//// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+using System;
+
+namespace Azure.Storage.Sas
+{
+    /// <summary>
+    /// A <see cref="BlobSasQueryParameters"/> object represents the components
+    /// that make up an Azure Storage Shared Access Signature's query
+    /// parameters.  You can construct a new instance using
+    /// <see cref="BlobSasBuilder"/>.
+    /// 
+    /// For more information, <see href="https://docs.microsoft.com/en-us/rest/api/storageservices/constructing-a-service-sas"/>.
+    /// </summary>
+    public sealed class BlobSasQueryParameters : SasQueryParameters
+    {
+        /// <summary>
+        /// Gets the Azure Active Directory object ID in GUID format.
+        /// </summary>
+        public new string KeyOid => base.KeyOid;
+
+        /// <summary>
+        /// Gets the Azure Active Directory tenant ID in GUID format
+        /// </summary>
+        public new string KeyTid => base.KeyTid;
```

Comment:
Is it too late to change the names of these properties to something more convenient for the customer? i.e. `KeyTenantId`?

Reply:
No, it's not, and you're right that we really need to make these more intelligible.  I'll fix this as part of the next round of API changes.

Reply:
Actually since it's just these two I'm fixing them now.  It's `KeyOid` to `KeyObjectId` and `KeyTid` to `KeyTenantId`.